<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddDataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Добавить данные</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>751</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="show_f_zb_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать таблицу с информацией о торгах</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_zb_Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>590</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать таблицу с информацией о фьючерсах</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterDataButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Фильтрация данных</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
